subprojects/parseq/src/main/java/com/linkedin/parseq/Engine.java

```diff
@@ -58,7 +58,7 @@
   private static final int DEFUALT_MAX_RELATIONSHIPS_PER_TRACE = 65536;
 
   public static final String MAX_CONCURRENT_PLANS = "_MaxConcurrentPlans_";
-  private static final int DEFUALT_MAX_CONCURRENT_PLANS = Integer.MAX_VALUE;
+  private static final int DEFAULT_MAX_CONCURRENT_PLANS = Integer.MAX_VALUE;
```

Comment:
Is this already in another PR?

Reply:
Deleted that PR. This should be the only one with this change now